cli/__snapshots__/cli_spec.js

```diff
@@ -354,3 +354,32 @@ exports['shows help for run --foo 1'] = `
   -------
   
 `
+
+exports['cli CYPRESS_ENV allows staging environment 1'] = `
+  code: 0
+  stderr:
+  -------
+  
+  -------
+
+`
+
+exports['cli CYPRESS_ENV catches environment "foo" 1'] = `
+  code: 11
+  stderr:
+  -------
+  Environment variable with reserved name "CYPRESS_ENV" was used.
+
+  Remove "CYPRESS_ENV" variable and run Cypress again.
```

Comment:
```suggestion
  Unset the "CYPRESS_ENV" environment variable and run Cypress again.
```

Reply:
I have updated the error message, are there other things to review?